client/containers/Landing/Landing.tsx

```diff
@@ -1,181 +1,158 @@
-		community: 'Academic Entrepreneurship'
-	},
-	{
-		title: 'Women of the Beat Generation',
-		date: 'Dec 14, 2021',
-		authors: 'Mary Paniccia Carden and Robert F. Barsky',
-		image: '/static/landing/_landing_pub2.png',
-		url: 'https://bandy-collection.pubpub.org/pub/o9gbzhf1/release/1',
-		desc: '',
-		doi: '',
-		collection: 'Collection Conversations',
-		community: 'W.T. Bandy Center for Baudelaire and Modern French Studies'
-	},
-	{
-		title: 'Differential Privacy and the 2020 US Census',
-		date: 'Jan 24, 2022',
-		authors: 'Simson Garfinkle',
-		image: '/static/landing/_landing_pub7.png',
-		url: 'https://doi.org/10.21428/2c646de5.7ec6ab93',
-		desc: '',
-		doi: 'https://doi.org/10.21428/2c646de5.7ec6ab93',
-		collection: 'Winter 2022',
-		community: 'MIT Case Studies in Social and Ethical Responsibilities of Computing'
-	},
-	{
-		title: 'Quare(-in) the Mainstream: Deconstructing New Media in Lil Nas X’s MONTERO',
-		date: 'Jul 12, 2022',
-		authors: 'Emily Thomas',
-		image: '/static/landing/_landing_pub8.png',
-		url: 'https://doi.org/10.21428/66f840a4.75cc622c',
-		desc: '',
-		doi: 'https://doi.org/10.21428/66f840a4.75cc622c',
-		collection: 'New Queer Cinema',
-		community: 'Sonic Scope'
-	}
-];
+type Props = {
+	featuredItems: LandingPageFeatures;
+};
 
-const communities = [
-	{
-		name: 'Harvard Data Science Review',
-		description: 'A microscopic, telescopic & kaleidoscopic view of data science.',
-		logo: '/static/landing/hdsr.png',
-		type: 'Journals',
-		category: 'Science',
-		link: 'https://hdsr.mitpress.mit.edu',
-	}
-];
+type FlattenedPub = {
+	pub: Pub;
+	title: React.ReactNode;
+	community: {
+		title: string;
+		url: string;
+	};
+	primaryCollection: null | {
+		title: string;
+		url: string;
+	};
+	publishedDate: null | string;
+	byline: React.ReactNode;
+};
 
-const Landing = () => {
-	const featureGrid = features.map((feature) => {
-		return (
-			<div className="feature" key={feature.icon}>
-				<Icon icon={feature.icon} className="icon" />
-				<div className="description">
-					<h4>{feature.title}</h4>
-					<p>{feature.desc}</p>
-				</div>
-			</div>
-		);
+const getFlattenedPubsFromFeaturedItems = (featuredItems: LandingPageFeatures): FlattenedPub[] => {
+	return featuredItems.pub.map((feature) => {
+		const { pub } = feature;
+		const { community } = pub;
+		const primaryCollection = getPrimaryCollection(pub.collectionPubs);
+		const publishedDate = getPubPublishedDate(pub);
+		return {
+			pub,
+			publishedDate: publishedDate ? formatDate(publishedDate) : null,
+			byline: <PubByline pubData={pub} />,
+			title: <PubTitle pubData={pub} shouldUseHtmlTitle />,
+			community: {
+				title: community.title,
+				url: communityUrl(community),
+			},
+			primaryCollection: primaryCollection
+				? {
+						title: primaryCollection.title,
+						url: collectionUrl(community, primaryCollection),
+				  }
+				: null,
+		};
 	});
+};
 
-	const pubList = pubs.map((pub) => {
+// const features = [
```

Comment:
I commented out some stuff here that's either unused or unfinished to make the linter happy and get a clean CI run.